docker/runtime/golang/kubeless.tpl.go

```diff
@@ -72,6 +72,16 @@ func handle(ctx context.Context, w http.ResponseWriter, r *http.Request) ([]byte
 }
 
 func handler(w http.ResponseWriter, r *http.Request) {
+
+	// add CORS support
+	w.Header().Set("Access-Control-Allow-Origin", "*")
+	w.Header().Set("Access-Control-Allow-Methods", "POST, GET, OPTIONS, PUT, DELETE")
```

Comment:
I would suggest to accept any request and let the user function deal with methods/headers. For doing so, and for consistency with the NodeJS approach, we can just return the same headers received. Also, I think we should just set these headers when the request is a CORS request:

```
w.Header().Set("Access-Control-Allow-Origin", "*")
if r.Method == "OPTIONS" {
  w.Header().Set("Access-Control-Allow-Methods", r.Header.Get("access-control-request-method"))
  w.Header().Set("Access-Control-Allow-Headers", r.Header.Get("access-control-request-headers"))
  return
}
```